Kitodo-DataManagement/src/main/resources/db/migration/V2_10__Change_visible_from_varchar_to_tinyint.sql

```diff
@@ -0,0 +1,23 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+--
+-- Migration: Change column visible in user table from varchar to tinyint
+
+-- 1. Update column visible to store correct tinyint values
+
+UPDATE user SET visible = '0' WHERE visible IS NULL;
```

Comment:
We can not exclude, that there MAY BE a change in the database. so we need to check for null, deleted and not deleted.